schemas/com.snowplowanalytics.snowplow/media_player_session/jsonschema/1-0-0

```diff
@@ -0,0 +1,102 @@
+{
+  "$schema": "http://iglucentral.com/schemas/com.snowplowanalytics.self-desc/schema/jsonschema/1-0-0#",
+  "description": "Schema for a context entity for media player events that tracks a session of a single media player usage",
+  "self": {
+    "vendor": "com.snowplowanalytics.snowplow",
+    "name": "media_player_session",
+    "format": "jsonschema",
+    "version": "1-0-0"
+  },
+  "type": "object",
+  "properties": {
+    "mediaSessionId": {
+      "type": "string",
+      "format": "uuid",
+      "description": "An identifier for the media session."
+    },
+    "startedAt": {
+      "type": [
+        "null",
+        "string"
+      ],
+      "description": "Date-time timestamp of when the session started.",
+      "format": "date-time"
+    },
+    "pingIterval": {
+      "type": [
+        "integer",
+        "null"
+      ],
+      "description": "Interval (seconds) in which the ping events will be sent. Default (specified in the tracker media docs) is assumed if not specified.",
+      "minimum": 0,
+      "maximum": 65535
+    },
+    "timePlayedContent": {
+      "type": [
+        "number",
+        "null"
+      ],
+      "description": "Total seconds user spent playing content (excluding ads).",
+      "minimum": 0,
+      "maximum": 2147483647
+    },
+    "timePlayedContentMuted": {
+      "type": [
+        "number",
+        "null"
+      ],
+      "description": "Total seconds user spent playing content on mute (excluding ads).",
+      "minimum": 0,
+      "maximum": 2147483647
+    },
+    "contentDurationPlayed": {
+      "type": [
+        "number",
+        "null"
+      ],
+      "description": "Total seconds of the content played. Each part of the content played is counted once (i.e., counts rewinding or rewatching the same content only once). Playback rate does not affect this value.",
```

Comment:
This is the only one that mentions playback rate, does that mean the other fields implicitly _are_ affected by playback rate?

Reply:
None of them are actually, but the other ones talk about the time user spent which I thought makes it more clear that playback rate can't affect it. On the other hand, "seconds of content played" sounds more unclear to me whether it can be affected by the playback rate. Happy to add the note to the other field descriptions if you think that would help.